packages/schema/docs/build/schema.md

```diff
@@ -1288,6 +1408,68 @@ List of before or after middlewares. Can be an array of functions or a single fu
 
 -----
 
+## /OutputFieldSchema
+
+Field schema specialized for output fields.
+
+#### Details
+
+* **Type** - `object`
+* [**Source Code**](https://github.com/zapier/zapier-platform/blob/zapier-platform-schema@16.2.0/packages/schema/lib/schemas/OutputFieldSchema.js)
+
+#### Properties
+
+Key | Required | Type | Description
+--- | -------- | ---- | -----------
+`key` | **yes** | `string` | A unique machine readable key for this value (IE: "fname").
+`label` | no | `string` | A human readable label for this value (IE: "First Name").
+`helpText` | no | `string` | A human readable description of this value (IE: "The first part of a full name."). You can use Markdown.
+`type` | no | `string` in (`'string'`, `'text'`, `'integer'`, `'number'`, `'boolean'`, `'datetime'`, `'file'`, `'password'`, `'copy'`, `'code'`) | The type of this value. Use `string` for basic text input, `text` for a large, `<textarea>` style box, and `code` for a `<textarea>` with a fixed-width font. Field type of `file` will accept either a file object or a string. If a URL is provided in the string, Zapier will automatically make a GET for that file. Otherwise, a .txt file will be generated.
```

Comment:
We should drop the types that make no sense for output fields, which at least are: `text` (as for output it's just about the value type, not the type of input field), `integer` (same), `copy` and `code`.

Not sue about `datetime` and `password`, but little harm in letting them be. And we'll need `file` for unfurl.